dist/gcc-compatible/lib_intrinsics.h

```diff
@@ -18,19 +18,33 @@
 
 #include "Hacl_IntTypes_Intrinsics.h"
 
-#define Lib_IntTypes_Intrinsics_add_carry_u32(x1, x2, x3, x4) \
-  (Hacl_IntTypes_Intrinsics_add_carry_u32(x1, x2, x3, x4))
+#if defined(HACL_CAN_COMPILE_VEC128)
```

Comment:
I don't think this is the right test. Based on the configure test, I would suggest adding after line 504 of the configure script:

```
else
  echo "#define HACL_CAN_COMPILE_UINT128 1" >> config.h
fi
```

and then using that